gui/qt/qrtextedit.py

```diff
@@ -48,6 +48,29 @@ def file_input(self):
         fileName, __ = QFileDialog.getOpenFileName(self, 'select file')
         if not fileName:
             return
+
+        image = QImage()
+        if image.load(fileName):
+            from electroncash.qrreaders import get_qr_reader
+            qr_reader = get_qr_reader()
```

Comment:
`get_qr_reader()` may return `None` on failure to find `libzbar`/`libosxqrdetect` here.  I know we are both super smart awesome geniuses that tried really hard to make that unlikely -- but it can still happen -- esp. if user runs from source on fresh clone.

Can you kindly do something here about that here?

Reply:
Oh right, forgot about that :) Will update.

Reply:
It might just be enough to say something like "image file was opened successfully but the QR detection library could not be loaded, please check that you have the library.. bla bla".. or even something shorter...